requirements.txt

```diff
@@ -1,34 +0,0 @@
-bleach==3.0.0
```

Comment:
why are you updating requirements? did you add any new dependency?

Reply:
Sir actually I moved Users to the top in order to use the created forgot password template. When it was at the bottom forgot password was actually using the custom django template for forgot password

Reply:
I didn't added any dependencies. There was a package called netifaces which wasn't installing itself. I realised that I can work without it and that is why I removed it during development and finally after doing everything I again added that package to requirements